src/requests/getImages.js

```diff
@@ -11,8 +11,10 @@ function getImages(query) {
         const imageResults = arrayResults.filter(
           (result) => result.data[0].media_type === "image"
         );
-        const parsedImages = imageResults.map((image) => image.links[0].href);
-        return parsedImages;
+        return imageResults.map((image) => [
+          image.links[0].href,
+          image.data[0].title,
```

Comment:
Nice! Though as mentioned above this should be an object rather than an array :)